src/SDKs/Search/DataPlane/Search.Tests/Tests/SkillsetsTests.cs

```diff
@@ -0,0 +1,610 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Microsoft.Azure.Search.Tests
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using Microsoft.Azure.Search.Models;
+    using Microsoft.Azure.Search.Tests.Utilities;
+    using Microsoft.Rest.Azure;
+    using Xunit;
+
+    public sealed class SkillsetsTests : SearchTestBase<SearchServiceFixture>
+    {
+        public const char PathSeparator = '/';
```

Comment:
nit: These constants shouldn't need to be public.

Reply:
fixed